addons/pos_hr/static/src/overrides/components/navbar/navbar.js

```diff
@@ -22,4 +22,18 @@ patch(Navbar.prototype, {
         this.pos.reset_cashier();
         this.pos.showScreen("LoginScreen");
     },
+    get showCreateProductButton() {
+        if (!this.pos.config.module_pos_hr || this.employeeIsAdmin()) {
+            return super.showCreateProductButton;
+        } else {
+            return false;
+        }
+    },
+    get showEditPlanButton() {
+        if (!this.pos.config.module_pos_hr || this.employeeIsAdmin()) {
+            return super.showEditPlanButton;
+        } else {
+            return false;
+        }
+    },
```

Comment:
There's a chance that `return super.showEditPlanButton;` will crash because `pos_restaurant` is not installed. Can you make sure to take that into account? I think using the `pos.config.module_pos_restaurant` flag is good enough to avoid the crash.